Replace gdebi with apt in project README

JJRcop opened the pull request:
Modern apt versions since apt 1.1 (2015) support local .deb installation if a full or relative path is provided, so gdebi is not required.

Also some minor grammatical corrections around the touched area

https://mvogt.wordpress.com/2015/11/30/apt-1-1-released/

kanishka-linux commented:
Hey that's great.. In fact, I'm myself using apt to install local packages, but did not update the README.
For installation on [raspberry-pi](https://github.com/kanishka-linux/rpi-utilities/blob/master/rpiscript.py#L43), I'm already using it :) 

If possible, can you also make changes [here](https://github.com/kanishka-linux/kawaii-player/blob/master/ubuntu/create_deb.py#L43) as well and test the deb file locally. You can add it as a separate commit, and if possible squash the two README commits into one, that will be helpful.

JJRcop commented:
Squashed README.md commits and added to `create_deb.py`.

Also modified the tail of create_deb.py to only print if `dpkg-deb` doesn't fail, and the `.deb file` exists.

kanishka-linux reviewed (commented):
added one minor comment on code, but forgot to submit :)

JJRcop commented on ubuntu/create_deb.py:
```diff
@@ -38,8 +38,12 @@
-	print('deb package created successfully in current directory. Now install the package using command: \n\nsudo gdebi {0}\n\n'.format(deb_pkg))
+	dpkg_errcode = subprocess.call(['dpkg-deb','--build',dest_dir])
+	deb_pkg = './'+os.path.basename(dest_dir)+'.deb'
+	if not dpkg_errcode:
```
I'll switch to `.exists()`, but otherwise I separated the ifs because the error message is if the file doesn't exist for some reason.

I originally made another error message if dpkg-deb failed for the if you see right now, but it seemed redundant because dpkg-deb already outputs an error, so create_deb also spitting out another error was pointless

JJRcop replied:
I added a dpkg-deb specific error to this if, I hope that resolves your concerns.

This means there are now two errors:
* dpkg-deb fails
* can't find .deb
